packages/shared/src/components/onboarding/OnboardingHeader.tsx

```diff
@@ -78,11 +79,15 @@ export const OnboardingHeader = ({
               </div>
             )}
           >
-            <Logo
-              logoClassName={{ container: 'h-6' }}
-              position={LogoPosition.Relative}
-              linkDisabled
-            />
+            {!isPlusStep ? (
+              <Logo
```

Comment:
The logo should only be different in onboarding pages not throughout the app?

Reply:
[I asked to Nimrod](https://dailydotdev.slack.com/archives/C08BCSCBD5F/p1739809376322079?thread_ts=1739447586.853689&cid=C08BCSCBD5F) and we would like to apply this change only in onboarding/plus page